tsdb/head.go

```diff
@@ -53,13 +53,23 @@ var (
 	// ErrAppenderClosed is returned if an appender has already be successfully
 	// rolled back or committed.
 	ErrAppenderClosed = errors.New("appender closed")
+
+	WalReplayStatus walReplayStatus
```

Comment:
See https://github.com/golang/go/wiki/CodeReviewComments#initialisms for capitalization of initialisms like WAL.